algorithms/ar-bsrh/python2/binary_search.py

```diff
@@ -0,0 +1,25 @@
+def binarysearch(arr,x):
+  low=0
+  high=len(arr)-1
+  mid=0
+  while(low<=high): 
```

Comment:
Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6277716)

Reply:
there is an unwanted white space at the end of line 5.